resin/models/device.py

```diff
@@ -60,10 +61,13 @@ def get(self, uuid):
             'eq': uuid
         }
         try:
-            return self.base_request.request(
+            devices = self.base_request.request(
                 'device', 'GET', params=params,
                 endpoint=self.settings.get('pine_endpoint')
-            )['d'][0]
+            )['d']
+            if len(devices) > 1:
```

Comment:
It can't be for the same reason as with apps -- app names are unique per user but device UUIDs are unique globally. I'm assuming the JS SDK has this check for code correctness since the API returns an array. IMO it's redundant, but I'm fine with it in any case.